docs/get-started.md

```diff
@@ -116,7 +105,145 @@ Execution succeeded (Time spent: 154/793 us)
 
 ---
 
-### Step 3 Simple Query Demo
+### From Source(Linux)
+
+**Prerequisite Tools**
+
+Nebula Graph is written in C++14, so it requires a complier supporting C++14 features.
+
+3rd-party Libraries
+
+-	autoconf
+-	automake
+-	libtool
+-	cmake
+-	bison
+-	unzip
+-	boost
+-	gperf
+-	krb5
+-	openssl
+-	libunwind
+-	ncurses
+-	readline
+
+It is recommended to install g++ 5 or higher Linux system, such as Fedora 29.
+Currently, we are using `git-lfs` to store the 3rd-party libraries so make sure
+`git-lfs` have been installed before fetching the source code.
+
+**Fetch from GitHub**
+
+```
+git clone https://github.com/vesoft-inc/nebula.git
+```
+
+**Compiling**
+
+```
+cmake ./
+```
+
+The default installation is in /usr/local path. To specify the installation path,
+use:
+
+```
+cmake -DCMAKE_INSTALL_PREFIX=$your_nebula_install_dir
+```
+
+to replace the `$your_nebula_install_dir` here
+
+Then run the following command:
+
+```
+make && make install
+```
+
+**Running**
+
+Configure nebula-metad.conf
+
+In your Nebula installation directory, run
+
+```
+cp etc/nebula-metad.conf.default etc/nebula-metad.conf
+```
+
+Modify configurations in nebula-metad.conf, such as
+
+- local_ip
+- port
+- ws_http_port metaservice HTTP
+- ws_h2_port metaservice HTTP2
+
+
+Configure nebula-storaged.conf
+
+```
+cp etc/nebula-storaged.conf.default etc/nebula-storaged.conf
+```
+
+Modify configurations in nebula-storaged.conf, such as
```

Comment:
@jude-zhu help to check:
if one begginer user only need to modify these four items, make it clear.  DO not use 'such as'. 'such as' mean more there are more than four items to modiry